include/internal/time.h

```diff
@@ -81,4 +87,45 @@ OSSL_TIME ossl_time_subtract(OSSL_TIME a, OSSL_TIME b)
     return err ? 0 : r;
 }
 
+/* Returns |a - b|. */
+static ossl_unused ossl_inline
+OSSL_TIME ossl_time_difference(OSSL_TIME a, OSSL_TIME b)
+{
+    return a > b ? ossl_time_subtract(a, b) : ossl_time_subtract(b, a);
+}
+
+static ossl_unused ossl_inline
+OSSL_TIME ossl_time_multiply(OSSL_TIME a, OSSL_TIME b)
```

Comment:
This looks dubious.  Multiplying a time by an int is reasonable.  But a time by a time???